mobify/platform-scaffold
# Adding support for variations on Merlin's

# Adding support for variations on Merlin's

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-88
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/669

## Changes
- Adds variation support on Merlin's Potions

## How to test-drive this PR
- Use Merlin's Connector
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to Eye of Newt, ensure variations work
- Use SFCC connector to ensure that the SFCC connector still works
